volumes/csi-wrapper/examples/aws/dynamic-provisioning/pod.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: app
+spec:
+  runtimeClassName: kata-remote
+  serviceAccountName: csi-ebs-podvm-sa
+  volumes:
+  - name: persistent-storage
+    persistentVolumeClaim:
+      claimName: ebs-claim
+  - name: kubelet-data-dir
+    hostPath:
+      path: /var/lib/kubelet
+      type: Directory
+  - emptyDir: {}
+    name: plugin-dir
+
+  containers:
+  - name: app
+    image: centos
+    command: ["/bin/sh"]
+    args: ["-c", "while true; do echo $(date -u) >> /data/out.txt; sleep 5; done"]
+    volumeMounts:
+    - name: persistent-storage
+      mountPath: /data
+
+  - name: csi-podvm-node-driver
+    env:
+      - name: KUBE_NODE_NAME
+        valueFrom:
+          fieldRef:
+            fieldPath: spec.nodeName
+      - name: CSI_NODE_NAME
+        valueFrom:
+          fieldRef:
+            fieldPath: spec.nodeName
+     # set when IMDS isn't accessible from the podvm
```

Comment:
remove this block?

Reply:
I'd leave it as IIUC in some cases IMDS might be disabled